Kernel/RCClauseStack.hpp

```diff
@@ -137,6 +137,11 @@ class RCClauseStack {
   
 private:
   ClauseStack _s;
+
+public:
+  ClauseStack& naked() {
```

Comment:
Is it worth the work? 

`RCClauseStack` is just a `ClauseStack` with reference counting on `push`/`pop`. It even brazenly allows `pushWithoutInc`/`popWithoutDec`. We are not using clause reference counting very well at the moment anywhere anyway. (I think a normal Stack with smart pointers to clauses would be right design to replace `RCClauseStack`). 

Given all this, I think I was even a bit enjoying breaking the interface of this already quite broken data structure... (Although, let's by all means discuss more if you think otherwise.)

Reply:
Yeah, looking more broadly at `RCClauseStack` it's already terrifying. Let's put this on the to-kill pile and move on, you're right.